pstack.yaml

```diff
@@ -79,4 +77,5 @@ update:
 
 test:
   pipeline:
-    - runs: /usr/bin/pstack --help
+    - name: Check pstack responds to --version
```

Comment:
Had hoped to add this also:

```bash
    - name: Check pstack can print the stack trace of a simple process
    - runs: |
        sleep 15 &
        sleep_pid=$!
        sudo pstack $sleep_pid > /tmp/pstack_output
        kill $sleep_pid

        if grep -q 'sleep' /tmp/pstack_output; then
          echo "pstack produced expected output."
        else
          echo "pstack did not produce expected output." 1>&2
          exit 1
        fi
```

But due to permissions issues, won't run in CI, even if attempting to run with sudo:

> trace of 28 failed: cannot open file '/proc/28/mem': Permission denied